### PR TITLE
Page crash fix

### LINKED ISSUE #97: [Bug] - Page crashes when trying to get next when no support items exist
### Pre-requisites:
- [x] I looked through both [open and closed issues](../issues?utf8=✓&q=is%3Aissue) and did not find an existing bug.

## Description

If you click get next when no items are available, the page throws an error

## Expected Behavior

The page displays that no requests are available

## Actual Behavior

The page throws and error and becomes unresponsive

## Steps to Reproduce

Load the homepage and hit the get next button with no items in the queue

### PR DESCRIPTION
**Pull request by Ulfingrad**
## Pre-Requisites
- [x] Yes, I updated [Authors.md](../blob/main/Authors.md) **OR** this is not my first contribution
- [x] Yes, I included and/or modified tests to cover relevent code **OR** my change is non-technical
- [x] Yes, I wrote this code entirely myself **OR** I properly attributed these changes in [Third Party Notices](../blob/main/THIRD-PARTY-NOTICES.txt)

## Description of Changes

Added a conditional to ensure that supportRequest's attributes are not accessed when null.

@Ulfingrad worked with Maha on this PR

## Related Issues

This change fixes #97 which was causing the admin page to crash when attempting to get the next item in queue when no such items exist.

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -210,9 +210,12 @@ const AdminPage: NextComponentType = () => {
...
                 <div className="alert alert-info mt-3">
-                  {lastPop.userNotified
-                    ? `${lastPop.supportRequest.name} has been notified to come over`
-                    : `There was a problem notifying ${lastPop.supportRequest.name}, send them a message and tell them you're ready for them`}
+                  {
+                  !(lastPop.supportRequest === null) ?  
+                    (lastPop.userNotified
+                      ? `${lastPop.supportRequest.name} has been notified to come over`
+                      : `There was a problem notifying ${lastPop.supportRequest.name}, send them a message and tell them you're ready for them`)
+                    : 'It appears you are trying to access a support request that does not exist!'}
```
Comment (SpencerKaiser) [inline src/pages/index.tsx:218]:
```suggestion
                  (lastPop.supportRequest !== null) ?  
                    (lastPop.userNotified
                      ? `${lastPop.supportRequest.name} has been notified to come over`
                      : `There was a problem notifying ${lastPop.supportRequest.name}, send them a message and tell them you're ready for them`)
                    : 'It appears you are trying to access a support request that does not exist!'}
```

**Review by SpencerKaiser — CHANGES_REQUESTED** [review]
When we chatted about this issue, @johnkahn made the suggestion to re-trigger the refresh early if this happens; can you add that functionality? Otherwise the button may still show items in the queue and they may click the button again.

**Comment by Ulfingrad** [comment]
@SpencerKaiser Yup, I'll get on those changes as soon as I can!

**Comment by SpencerKaiser** [comment]
@Ulfingrad remember to reach out to @johnkahn if you have any questions about the final piece of this PR

File: src/pages/index.tsx [hunk 2]
```diff
@@ -39,12 +39,26 @@ const AdminPage: NextComponentType = () => {
...
+    }
+  }
+
   React.useEffect(() => {
     formik.setFieldValue('adminName', window.localStorage.getItem(ADMIN_NAME_KEY));
 
     const timer = setInterval(() => {
       setLastUpdateEpoch(Date.now());
-    }, 30000);
+    }, 3000);
```
Comment (SpencerKaiser) [inline src/pages/index.tsx:61]: 3 seconds is too fast, let's settle for a good middle ground of 10s
```suggestion
    }, 10000);
```

**Review by SpencerKaiser — APPROVED** [review]
Great work you two! @johnkahn is going to review here shortly and he'll merge unless he has other changes 🎉

**Review by johnkahn — APPROVED** [review]

**Pull request merged**